src/App.js

```diff
@@ -1,18 +1,24 @@
 import React, { Component } from 'react';
 import logo from './logo.svg';
+import FormSubmit from './form/form-submit'
 import './App.css';
 
 class App extends Component {
```

Comment:
Since this component is stateless and only calls the render function, you can use an arrow function instead of a class.
This should look something like one of the two options:
```
const App = () => {
    return (
        ...
    )
}
```
```
const App = () => (
    ...
)
```